pennylane/devices/tests/test_compare_default_qubit.py

```diff
@@ -68,6 +68,40 @@ def circuit(theta, phi):
         assert np.allclose(qnode(theta, phi), qnode_def(theta, phi), atol=tol(dev.shots))
         assert np.allclose(grad(theta, phi), grad_def(theta, phi), atol=tol(dev.shots))
 
+    def test_projector_expectation(self, device, tol):
+        """Test that arbitrary multi-mode Projector expectation values are correct"""
+        n_wires = 2
+        dev = device(n_wires)
+        dev_def = qml.device("default.qubit", wires=n_wires)
+
+        if dev.shots is not None:
+            pytest.skip("Device is in non-analytical mode.")
+
+        if "Projector" not in dev.observables:
+            pytest.skip("Device does not support the Projector observable.")
+
+        if dev.name == dev_def.name:
+            pytest.skip("Device is default.qubit.")
```

Comment:
How come you are skipping `default.qubit`?

Reply:
Oh nevermind, I just realized because this test compares to `default_qubit`, so it would just be comparing to itself 😆